app/views/account/IsraelSignupView.coffee

```diff
@@ -0,0 +1,142 @@
+RootView = require 'views/core/RootView'
+template = require 'templates/account/israel-signup-view'
+forms = require 'core/forms'
+errors = require 'core/errors'
+utils = require 'core/utils'
+User = require 'models/User'
+
+class AbortError extends Error
+
+formSchema =
+  type: 'object'
+  properties: _.pick(User.schema.properties, 'name', 'password')
+  required: ['name', 'password']
+
+
+module.exports = class IsraelSignupView extends RootView
+  id: 'israel-signup-view'
+  template: template
+  
+  events:
+    'submit form': 'onSubmitForm'
+    'input input[name="name"]': 'onChangeName'
+    'input input[name="password"]': 'onChangePassword'
+    
+  initialize: ->
+    @state = new Backbone.Model({
+      fatalError: null
+      # Possible errors:
+      #   'signed-in': They are logged in
+      #   'missing-input': Required query parameters are not provided
+      #   'email-exists': Given email exists in our system
+      #   Any other string will be shown directly to user
+      
+      formError: null
+      loading: true
+      submitting: false
+      queryParams: _.pick(utils.getQueryVariables(),
+        'il_id'
+        'first_name'
+        'last_name'
+        'email'
+        'school'
+        'city'
+        'state'
+        'district'
+      )
+      name: ''
+      password: ''
+    })
+    
+    { il_id, email } = @state.get('queryParams')
+    
+    # sanity checks
+    if not me.isAnonymous()
+      @state.set({fatalError: 'signed-in', loading: false})
+    
+    else if not (il_id and email) or not forms.validateEmail(email)
+      @state.set({fatalError: 'missing-input', loading: false})
+      
+    else
+      User.checkEmailExists(email)
+      .then ({exists}) =>
+        @state.set({loading: false})
+        if exists
+          @state.set({fatalError: 'email-exists'})
+          
+      .catch =>
+        @state.set({fatalError: $.i18n.t('loading_error.unknown'), loading: false})
+        
+    @listenTo(@state, 'change', _.debounce(@render))
+
+  getRenderData: ->
+    c = super()
+    return _.extend({}, @state.attributes, c)
+    
+  onChangeName: (e) ->
+    # sync form info with state, but do not re-render
+    @state.set({name: $(e.currentTarget).val()}, {silent: true})
+    
+  onChangePassword: (e) ->
+    @state.set({password: $(e.currentTarget).val()}, {silent: true})
+
+  displayFormSubmitting: ->
+    @$('#create-account-btn').text($.i18n.t('signup.creating')).attr('disabled', true)
+    @$('input').attr('disabled', true)
+
+  displayFormStandingBy: ->
+    @$('#create-account-btn').text($.i18n.t('login.sign_up')).attr('disabled', false)
+    @$('input').attr('disabled', false)
```

Comment:
*points out that we're mixing jquery-dom-manipulation and state-based rendering*. I assume this is because rendering interacts badly with the form inputs? Or maybe that this is just easier/lower overhead?